src/ddmd/backend/backconfig.c

```diff
@@ -51,7 +51,7 @@ void out_config_init(
                         // 2: fake it with C symbolic debug info
         bool alwaysframe,       // always create standard function frame
         bool stackstomp,        // add stack stomping code
-        bool avx,               // use AVX instruction set
+        unsigned char avx,      // use AVX instruction set (0, 1, 2)
```

Comment:
No, because DMC++ is not C++11, and this is such a minor thing as to not justify requiring C++11.